.github/workflows/build_hub.yml

```diff
@@ -22,7 +20,7 @@ jobs:
       has_router_change: ${{ steps.build_vars.outputs.has_router_change }}
     steps:
       - name: "Check out changes"
-        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # This is v4.1.1
```

Comment:
No need for the comment, Dependabot is likely to replace the version "soon" anyways.